### PR TITLE
Add ROT space

### PR DESCRIPTION
**Pull request by wallbanger1**
This should be ready to go. Let me know if there are any errors. This was done at the request of Tim the founding dev for RottenSwap(ROT). https://github.com/rottenswap

### CODE REVIEW

File: spaces/rottenswap/index.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "key": "rotten-swap",
```
Comment (e-asia) [inline spaces/rottenswap/index.json:2]: The key of your space must also be the folder name and skin class name.

File: spaces/rottenswap/index.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "symbol": "ROT",
+  "name": "RottenSwap",
+  "defaultView": "core",
+  "address": "0xd04785c4d8195e4a54d9dec3a9043872875ae9e2",
+  "token": "0xd04785c4d8195e4a54d9dec3a9043872875ae9e2",
+  "core": [],
+  "min": 30000,
+  "invalid": []
+}
```
Comment (e-asia) [inline spaces/rottenswap/index.json:11]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/rottenswap/index.json (hunk 2 above)
Comment (wallbanger1) [reply spaces/rottenswap/index.json:13]: Ok this has been updated. I didn't see those documented anywhere. It is not mentioned on ur linked docs.
image

File: spaces/rottenswap/index.json (hunk 1 above)
Comment (wallbanger1) [reply spaces/rottenswap/index.json:2]: Noted and fixed. I changed the folder to include the hyphen. The class name is the same as well.

**Comment by wallbanger1** [comment]
Ok I think everything is updated now. Please let me know if there are any other issues. Thanks!

**Review by e-asia — APPROVED** [review]

**Pull request merged**